tests/FSharp.Test.Utilities/TestFramework.fs

```diff
@@ -463,14 +478,24 @@ let initializeSuite () =
 
 let suiteHelpers = lazy (initializeSuite ())
 
-let testConfig (testDir: string) =
+let testConfig sourceDir (relativePathToTestFixture: string) =
+    let cfg = suiteHelpers.Value
+    let testFixtureFullPath = Path.GetFullPath(sourceDir ++ relativePathToTestFixture)
+
+    let description = relativePathToTestFixture.Split('\\', '/') |> String.concat "-"
+
+    let tempTestRoot = createTemporaryDirectory description
+    let tempTestDir =
+        DirectoryInfo(tempTestRoot)
+            .CreateSubdirectory(relativePathToTestFixture)
+            .FullName
+    copyDirectory testFixtureFullPath tempTestDir true
```

Comment:
Also, for my understanding here - why is this copying needed, as in, why (or in which cases) cannot things be created at the temp dir right away?

Reply:
What we copy is the text fixture from the repo, that can look like this:
https://github.com/dotnet/fsharp/tree/main/tests/fsharp/core/attributes
The test that would (on main) run directly in the repo directory, now runs in temp, seeing the same structure of dirs/files and often writing to it.
I'm not sure I get what you mean by right away?

Reply:
Oh right, that makes sense, it was more about "what is a fixture here" then. Yeah that's okay, I am just trying to understand how things are working here :)